src/earn/prepareTransactions.ts

```diff
@@ -118,100 +123,169 @@ export async function prepareWithdrawAndClaimTransactions({
       ],
       ...shortcutTriggerArgs?.withdraw,
     })
+  Logger.debug(TAG, 'prepareWithdrawTransactions', {
```

Comment:
nit: Update this fn name in the log to match the actual fn name

Reply:
Fixed in 501c1c34409b34d4835f919c48cb863eb29ae49d!